<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArtifactEditClass</class>
 <widget class="QDialog" name="ArtifactEditClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ArtifactEdit</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QToolButton" name="addInstanceToolButton">
     <property name="text">
      <string>AddInstance</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="delInstanceToolButton">
     <property name="text">
      <string>DelInstance</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTreeView" name="treeView">
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>addInstanceToolButton</sender>
   <signal>clicked()</signal>
   <receiver>ArtifactEditClass</receiver>
   <slot>actAddInstance()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delInstanceToolButton</sender>
   <signal>clicked()</signal>
   <receiver>ArtifactEditClass</receiver>
   <slot>actDelInstance()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ArtifactEditClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ArtifactEditClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>actAddInstance()</slot>
  <slot>actDelInstance()</slot>
 </slots>
</ui>
